otel-extension/src/test/scala/io/scalac/mesmer/agent/utils/InstallAgent.scala

```diff
@@ -0,0 +1,54 @@
+package io.scalac.mesmer.agent.utils
+
+import net.bytebuddy.ByteBuddy
+import net.bytebuddy.agent.ByteBuddyAgent
+import net.bytebuddy.agent.builder.AgentBuilder
+import net.bytebuddy.agent.builder.AgentBuilder.TypeStrategy
+import net.bytebuddy.dynamic.scaffold.TypeValidation
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.TestSuite
+
+import io.scalac.mesmer.agent.Agent
+import io.scalac.mesmer.agent.util.i13n.InstrumentModuleFactory
+import io.scalac.mesmer.agent.utils.InstallAgent.allInstrumentations
+import io.scalac.mesmer.core.module.MesmerModule
+import io.scalac.mesmer.otelextension.instrumentations.akka.stream.AkkaStreamAgent
+
+object InstallAgent {
+
+  def allInstrumentations: Agent = AkkaStreamAgent.agent
+}
+
+abstract class InstallAgent extends TestSuite with BeforeAndAfterAll {
+
+  protected def agent: Agent = allInstrumentations
+
+  private val builder = new AgentBuilder.Default(
+    new ByteBuddy()
+      .`with`(TypeValidation.DISABLED)
+  )
+    .`with`(AgentBuilder.RedefinitionStrategy.REDEFINITION)
+    .`with`(TypeStrategy.Default.REDEFINE)
```

Comment:
That's changed needleslly. I will revert it back to RETRANSFORMATION. The tests will still pass